src/DownloadCommand.php

```diff
@@ -32,29 +32,31 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->loadConfig($input);
 
         $vcards = array();
-
-        $progress = new ProgressBar($output);
+        if ($input->getOption('image')) {
```

Comment:
```
$substitutes = ($input->getOption('image')) ? ['PHOTO'] : [];

Reply:
Ich nehme an, das ersetzt das if-than-else in einem Rutsch.
Ist viel eleganter, aber ich wäre nicht darauf gekommen (Goldschmied vs. Hufschmied)
Wieder was gelernt
Habe es in RunCommand auch gleich ersetzt

Reply:
Genau- das ist ein "ternary" assignment